server/__main__.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI, WebSocket
+from starlette.staticfiles import StaticFiles
+
+"""
+uvicorn server.__main__:app --reload
+
+Run FastAPI server from both web and websocket"""
+app = FastAPI()
```

Comment:
```suggestion
"""
uvicorn server.__main__:app --reload

Run FastAPI server from both web and websocket
"""
app = FastAPI()
```
I think this looks a bit better

Reply:
Done and removed 'web'.